src/prefect/environments/storage/docker.py

```diff
@@ -85,11 +87,12 @@ def __init__(
         image_tag: str = None,
         env_vars: dict = None,
         files: dict = None,
-        base_url: str = None,
         prefect_version: str = None,
         local_image: bool = False,
         ignore_healthchecks: bool = False,
         secrets: List[str] = None,
+        base_url: str = None,
+        tls_config: "docker.tls.TLSConfig" = None,
```

Comment:
should this default to False to match the underlying API call's type and default?

Reply:
It's kind of a funny kwarg because it can both be `bool` and `TLSConfig`. In my testing setting `TLSConfig` as None has the same effect as using `False` so I would prefer to not mix the two

Reply:
Actually disregard. In order to match expectations from the Docker library I am going to match on both types